Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -564,6 +564,8 @@ def panorama_commit(args):
         'cmd': f'<commit>{command}</commit>',
         'key': API_KEY
     }
+    if target := args.get('target', None):
```

Comment:
```suggestion
    if target := args.get('target'):
```

get method of a dict returns ```None``` by default.
not super important change though, the way you did is good too :)